packages/core/src/sql-on-fhir/eval.ts

```diff
@@ -0,0 +1,243 @@
+import { Resource, ViewDefinition, ViewDefinitionSelect } from '@medplum/fhirtypes';
```

Comment:
I got these types working on a branch locally too. Curious how you approached it (it's not committed in the PR). I loaded up the structure definitions manually into @medplum/generator from the IG's package.

This is actually something that's generally _very_ useful and under-explored in TypeScript FHIR support - typings for profiles.

Reply:
Yes, sorry for confusion, the types were added in separate PR: https://github.com/medplum/medplum/pull/4674

For lack of a better place, we stuffed them in `packages/definitions/dist/fhir/r4/profiles-medplum.json` for now.